### PR TITLE
[RFR] Removed web_ui toolbars from test_default_views_infra

### PR DESCRIPTION
**Pull request by ohiliazo**
__Updating views__ to have them toolbar included.
__Updating tests__ to remove web_ui toolbars.

### CODE REVIEW

File: cfme/tests/configure/test_default_views_infra.py [hunk 1]
```diff
@@ -36,9 +36,10 @@ def set_and_test_default_view(group_name, view, page):
     dest = 'All'
     if group_name == 'VMs':
         dest = 'VMsOnly'
-    navigate_to(page, dest, use_resetter=False)
-
-    assert tb.is_active(view), "{} view setting failed".format(view)
+    this_view = navigate_to(page, dest, use_resetter=False)
+    print(this_view.toolbar)
```
Comment (psav) [inline cfme/tests/configure/test_default_views_infra.py]: **REQUIRED:** Remove print :)

**Comment by dajoRH** [comment]
Would you mind rebasing this Pull Request against latest master, please? :trollface:
*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 6defaec529b9c3e2ae7dc0ef78e3a47ad8d0fbd5:

`cfme/services/myservice/ui.py`:
- :red_circle: Line 38:11: **N801** *class names should use CapWords convention*

`cfme/services/workloads.py`:
- :red_circle: Line 54:11: **N801** *class names should use CapWords convention*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

**Comment by dajoRH** [comment]
Lint report for commit 5930a3f4863a00fc06c6e146f60a2e0adedc4936:

`cfme/services/myservice/ui.py`:
- :red_circle: Line 38:11: **N801** *class names should use CapWords convention*

`cfme/services/workloads.py`:
- :red_circle: Line 54:11: **N801** *class names should use CapWords convention*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

**Comment by ohiliazo** [comment]
@dajoRH toolbar is lowercase because every other nested classed are lowercase.

**Comment by dajoRH** [comment]
Lint report for commit 9cef0bc18ab333c79dc1f703a4226c6828adb6ff:

`cfme/services/myservice/ui.py`:
- :red_circle: Line 38:11: **N801** *class names should use CapWords convention*

`cfme/services/workloads.py`:
- :red_circle: Line 54:11: **N801** *class names should use CapWords convention*

Please, rectify these issues :smirk: .


**No commit flaws detected**.

*CFME QE Bot*

**Pull request merged**